modules/ding_nodelist/plugins/content_types/ding_nodelist.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @file
+ * Ctools plugin definition file.
+ */
+
+/**
+ * General ctools plugin settings.
+ */
+$plugin = array(
+  'title' => t('Nodelist'),
+  'category' => t('Ding!'),
+  'description' => t('Allows to create highliy customizeable list of contents'),
+  'single' => TRUE,
+  'content_types' => 'ding_nodelist_content_type',
+  'render callback' => 'ding_nodelist_content_type_render',
+  'admin title' => 'ding_nodelist_content_type_admin_title',
+  'admin info' => 'ding_nodelist_content_type_admin_info',
+  'edit form' => 'ding_nodelist_content_type_edit_form',
```

Comment:
Please move the functions related to the the plugin to this file unless there is a specific reason to keep them in the `.module` file.